masonry/src/contexts.rs

```diff
@@ -540,11 +488,34 @@ impl_context_method!(WidgetCtx<'_>, EventCtx<'_>, LifeCycleCtx<'_>, {
 // --- MARK: OTHER METHODS ---
 // Methods on all context types except PaintCtx and AccessCtx
 impl_context_method!(
-    WidgetCtx<'_>,
+    MutateCtx<'_>,
     EventCtx<'_>,
     LifeCycleCtx<'_>,
     LayoutCtx<'_>,
     {
+        pub fn mutate_self_later(
```

Comment:
Would be good to have a doc comment for this method and the one below. Probably a pretty short one.